### PR TITLE
Update all deps, update to Symfony 5.4

### PR DESCRIPTION
**Pull request by Seldaek**

### CODE REVIEW

File: src/Entity/Version.php [hunk 1]
```diff
@@ -154,47 +155,49 @@ class Version
     /**
      * @ORM\Column(type="text", nullable=true)
      */
-    private $source;
+    private string|null $source;
```
Comment (stof) [inline src/Entity/Version.php:158]: should probably have `= null` for all nullable fields

Comment (Seldaek) [reply src/Entity/Version.php:158]: Right, I always forget those..

BTW related to this, do you know if it's safe to type the OneToMany & co as `private ArrayCollection $foo` or is there a case where doctrine will initialize with something that's not compatible?

I guess the most accurate type would be `Doctrine\Common\Collections\Collection & Doctrine\Common\Collections\Selectable` but that'll require 8.1.

Comment (stof) [reply src/Entity/Version.php:158]: When loading from the DB, the ORM will use a PersistentCollection, not an ArrayCollection.

you need to use the Collection interface as type, not an implementation of it.

Comment (Seldaek) [reply src/Entity/Version.php]: Alright, hopefully the latest iteration is correct. All properties of Version are typed now, PHPStan is happy, we'll need to see if prod agrees 😄

**Pull request merged**